hq/app/schedule/Dynamo.scala

```diff
@@ -0,0 +1,47 @@
+package schedule
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
+import com.amazonaws.services.dynamodbv2.model.{AttributeValue, GetItemRequest, PutItemRequest}
+import model.{IamAuditEmail, IamAuditUser}
+import play.api.Logging
+
+import scala.collection.JavaConverters._
+import scala.util.control.NonFatal
+
+trait AttributeValues {
+  def S(str: String) = new AttributeValue().withS(str)
+  def L(list: List[AttributeValue]) = new AttributeValue().withL(list.asJava)
+  def N(number: Long) = new AttributeValue().withN(number.toString)
+  def N(number: Double) = new AttributeValue().withN(number.toString)
+  def B(boolean: Boolean) = new AttributeValue().withBOOL(boolean)
+  def M(map: Map[String,  AttributeValue]) = new AttributeValue().withM(map.asJava)
+}
+
+class Dynamo(client: AmazonDynamoDB, tableName: String) extends AttributeValues with Logging {
+
+  def get(key: Map[String, AttributeValue]): Option[Map[String, AttributeValue]] = Option(client.getItem(
+    new GetItemRequest().withTableName(tableName).withKey(key.asJava)).getItem) map (_.asScala.toMap)
+
+  def put(item: Map[String, AttributeValue]): Unit = try {
+    client.putItem(
+      new PutItemRequest().withTableName(tableName).withItem(item.asJava))
+  } catch {
+    case NonFatal(e) =>
+    logger.error(s"unable to put item to dynamoDB table: ${e.getMessage}", e)
+  }
+
+  def emailToMap(e: IamAuditEmail): Map[String, AttributeValue] = {
```

Comment:
I think we're using a mix of 'email' and 'alert' to refer to the same thing, it might be good where possible to reduce this to just use 'alert' to simplify naming a bit